package.json

```diff
@@ -3,30 +3,31 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "tailwind": "npx tailwindcss -i ./styles/input.css -o ./dist/output.css --watch",
     "dev": "next dev",
-    "start": "run-p tailwind dev",
     "build": "next build",
     "lint": "next lint"
   },
   "dependencies": {
     "@emailjs/browser": "^3.11.0",
+    "@tailwindcss/postcss": "^4.1.11",
     "@types/node": "20.4.2",
-    "@types/react": "18.2.15",
-    "@types/react-dom": "18.2.7",
+    "@types/react": "19.1.8",
+    "@types/react-dom": "19.1.6",
     "autoprefixer": "^10.4.14",
     "axios": "^1.4.0",
     "emailjs-com": "^3.2.0",
     "eslint": "8.45.0",
-    "eslint-config-next": "13.4.10",
-    "next": "^13.2.4",
-    "postcss": "8.4.26",
-    "react": "18.2.0",
-    "react-dom": "18.2.0",
+    "eslint-config-next": "15.3.5",
+    "next": "15.3.5",
+    "postcss": "^8.4.26",
+    "radix-ui": "^1.4.2",
```

Comment:
FYI there is a controversy around Radix. The library is getting less maintenance after WorkOS acquisition.
We can use other libraries if we don't feel comfortable using Radix.